src/views/ActorPageView.vue

```diff
@@ -3,23 +3,31 @@
     <div class="d-flex actor-details">
       <div class="actor-details-left-side d-flex flex-column">
         <div class="actor-details-left-side__img">
-          <img :src="actorDetails.profile_path ? 'https://image.tmdb.org/t/p/original' + actorDetails.profile_path : 'https://st4.depositphotos.com/17828278/24401/v/600/depositphotos_244011872-stock-illustration-image-vector-symbol-missing-available.jpg'"/>
+          <img
+              :src="actorDetails.profile_path ? 'https://image.tmdb.org/t/p/original' + actorDetails.profile_path : 'https://st4.depositphotos.com/17828278/24401/v/600/depositphotos_244011872-stock-illustration-image-vector-symbol-missing-available.jpg'"/>
         </div>
         <div class="actor-details-left-side__personal-info">
-          <h4>Personal Info</h4>
           <div class="personal-info__details">
-            <div class="details__item"><h5>Known For</h5><p>{{actorDetails.known_for_department}}</p></div>
-            <div class="details__item"><h5>Gender</h5><p>{{getGender()}}</p></div>
-            <div class="details__item"><h5>Birthday</h5><p>{{!actorDetails.birthday ? '' : getAge()}}</p></div>
-            <div class="details__item"><h5>Place of Birth</h5><p>{{actorDetails.place_of_birth}}</p></div>
-            <div class="details__item"><h5>Also Known As</h5><p>{{!actorDetails.also_known_as ? actorDetails.name : actorDetails.also_known_as[0]}}</p></div>
+            <div class="details__item"><h5 if="actorDetails.known_for_department">Known For</h5>
+              <p>{{ actorDetails.known_for_department }}</p></div>
+            <div class="details__item"><h5 v-if="actorDetails.gender">Gender</h5>
+              <p>{{ getGender() }}</p></div>
+            <div class="details__item"><h5 v-if="actorDetails.birthday">Birthday</h5>
+              <p>{{ !actorDetails.birthday ? '' : getAge() }}</p></div>
+            <div class="details__item"><h5 v-if="actorDetails.place_of_birth">Place of Birth</h5>
+              <p>{{ actorDetails.place_of_birth }}</p></div>
+            <div class="details__item"><h5
+                v-if="actorDetails.also_known_as ? actorDetails.also_known_as.length !== 0 : false">Also Known As</h5>
+              <p>{{ !actorDetails.also_known_as ? actorDetails.name : actorDetails.also_known_as[0] }}</p></div>
           </div>
         </div>
       </div>
       <div class="actor-details-right-side">
-        <h1>{{actorDetails.name}}</h1>
-        <h2>Biography</h2>
-        <p>{{actorDetails.biography}}</p>
+        <h1>{{ actorDetails.name }}</h1>
+        <h2 v-if="actorDetails.biography">Biography</h2>
+        <p v-if="actorDetails.biography">{{ actorDetails.biography }}</p>
+        <h2 v-if="actorKnownFor">Known for</h2>
+        <known-for-list/>
```

Comment:
For component like this, it's better to make them 'dumb', so that they display what is given to them by props, instead of binding them to the VueX store values internally.